forms-flow-web/src/components/CustomComponents/SortableHeader.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation  } from "react-i18next";
+
+const SortableHeader = ({ columnKey, title, currentSort, handleSort,className }) => {
+    const { t } = useTranslation();
+    const isSorted = currentSort.sortBy === columnKey;
+    const sortedOrder = isSorted ? currentSort.sortOrder : "asc";
+    const handleKeyDown = (event)=>{
+      if (event.key === 'Enter') {  
+        handleSort(columnKey);
+        }
+    };
+    return (
+      <div
+        className= {`d-flex align-items-center justify-content-between cursor-pointer ${className || ''}`}
```

Comment:
instead of doing className || "" you can give default value to className="" in props

Reply:
Updated.